cwltool/load_tool.py

```diff
@@ -48,7 +48,7 @@ def fetch_document(argsworkflow,  # type: Union[Text, Dict[Text, Any]]
     workflowobj = None  # type: CommentedMap
     if isinstance(argsworkflow, string_types):
         split = urllib.parse.urlsplit(argsworkflow)
-        if split.scheme:
+        if split.scheme and split.scheme in [u'http',u'https',u'file']:
```

Comment:
This is interesting. On windows path Drive letters get considered as scheme. ex `E:\foo` here on urlsplit E becomes scheme. I complained about this to urllib people, but it turns out that is there implementation as use `:` to find scheme